frontend/src/container/PipelinePage/ListOfPipelines.tsx

```diff
@@ -0,0 +1,361 @@
+/* eslint-disable react/no-unstable-nested-components */
+import {
+	CopyFilled,
+	DeleteFilled,
+	DownOutlined,
+	EditOutlined,
+	ExclamationCircleOutlined,
+	EyeFilled,
+	HolderOutlined,
+	PlusCircleOutlined,
+	PlusOutlined,
+	RightOutlined,
+} from '@ant-design/icons';
+import {
+	Avatar,
+	Button,
+	List,
+	Modal,
+	Space,
+	Switch,
+	Table,
+	Tag,
+	Typography,
+} from 'antd';
+import { ColumnProps } from 'antd/es/table';
+import { ColumnsType } from 'antd/lib/table';
+import DraggableTableRow from 'components/DraggableTableRow';
+import useComponentPermission from 'hooks/useComponentPermission';
+import React, { useCallback, useEffect, useState } from 'react';
+import update from 'react-addons-update';
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
+import { useSelector } from 'react-redux';
+import { AppState } from 'store/reducers';
+import AppReducer from 'types/reducer/app';
+
+import {
+	deleteModalDescriptionStyle,
+	iconStyle,
```

Comment:
why so much style ?

Reply:
I am using external css instead of inline css. even if we add only one css.
@palashgdev